R/test-threads.R

```diff
@@ -300,62 +300,120 @@ delete_models <- function(.mods, .tags = "test threads", .force = FALSE){
 #' Parse model files and overwrite estimation iterations
 #'
 #' @param .mods a list of model objects. Generally created by `test_threads()`.
-#' @param .max_eval Max number of iterations for NONMEM to run.
-#' @param .detect Estimation options that will be set to `.max_eval` if found.
+#' @param .cap_iterations Max number of iterations for NONMEM to run.
 #'
 #' @importFrom readr parse_number
 #' @importFrom checkmate assert_int
+#' @importFrom stringr str_detect
 #' @importFrom rlang is_empty
 #'
 #' @keywords internal
-adjust_estimation_options <- function(.mods, .max_eval, .detect = c("MAXEVAL", "NITER", "NBURN", "MAX")){
+adjust_estimation_options <- function(.mods, .cap_iterations){
 
-  assert_true(all(.detect %in% c("MAXEVAL", "NITER", "NBURN", "MAX")))
-
-  if(!is.null(.max_eval)){
-    assert_int(.max_eval, lower = 1)
+  if(!is.null(.cap_iterations)){
+    assert_int(.cap_iterations, lower = 1)
+    .detect = c("MAXEVAL", "NITER", "NBURN", "MAX")
     search_str <- paste(.detect, collapse = "|")
     map(.mods, function(.mod){
+
```

Comment:
`.detect` and `search_str` ([these lines](https://github.com/metrumresearchgroup/bbr/blob/f6670d88c6a1e7fe73c4cbaa6dd78829eeb11ccb/R/test-threads.R#L315-L316)) can be deleted because they're no longer used.